### PR TITLE
Add rake-nltk

### PR DESCRIPTION
**Pull request by cu8blank**
I would like to add the rake-nltk Python library into conda-forge.

### CODE REVIEW

File: recipes/rake-nltk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - nltk
```
Comment (sodre) [inline recipes/rake-nltk/meta.yaml]: Is nltk really a build-time dependency or just runtime?

File: recipes/rake-nltk/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'RAKE short for Rapid Automatic Keyword Extraction algorithm'
+
+  description: |
+    RAKE short for Rapid Automatic Keyword Extraction algorithm, is a domain independent
+    keyword extraction algorithm which tries to determine key phrases in a body of text
+    by analyzing the frequency of word appearance and its co-occurance with other words
+    in the text.
+  doc_url: https://pypi.org/project/rake-nltk/#files
```
Comment (sodre) [inline recipes/rake-nltk/meta.yaml]: This is not a good "doc_url". It is okay to leave it blank if no docs are available.

File: recipes/rake-nltk/meta.yaml (hunk 1 above)
Comment (cu8blank) [reply recipes/rake-nltk/meta.yaml]: nltk is only a runtime dependency. I removed it from the meta.yaml file.

File: recipes/rake-nltk/meta.yaml (hunk 2 above)
Comment (cu8blank) [reply recipes/rake-nltk/meta.yaml]: I couldn't find a good doc_url so I deleted this line from the meta.yaml file.

**Comment by sodre** [comment]
@jakirkham,

How should we handle packages that after pip install downloads third-party datasets? In our case, [rake-nltk/setup.py L15-L20](https://github.com/csurfer/rake-nltk/blob/0f6988223bed9fc1d91020f7691617edee36ca07/setup.py#L15-L20) post-install code downloads nltk data.

**Comment by jakirkham** [comment]
We ended up bundling the NLTK data in a package. So maybe it can just be added as a requirement to bypass the download? Hopefully that solves this particular problem.

Generally packaging data has been done in a few cases. We might want to move to using split packages to solve this problem in the future instead of having separate feedstocks. For example, building `nltk` would produce and `nltk_data` package and an `nltk` package. Something to think about.

ref: https://github.com/conda-forge/nltk_data-feedstock

File: recipes/rake-nltk/meta.yaml (hunk 1 above)
Comment (sodre) [reply recipes/rake-nltk/meta.yaml]: @cu8blank, 
Please add nltk_data as a build-time and runtime dependency. This should fix the download issue as explained by @jakirkham below.

Once that is fixed the only other thing I can think about is to check if rake can be bundled as a noarch package..

**Pull request merged**